### PR TITLE
Fix reserved words matching prefixes of field names

### PR DESCRIPTION
**Pull request by nicpottier**
We weren't defining the lexer quite right for reserved words..

See "specification of tokens" here:
  http://www.dabeaz.com/ply/ply.html#ply_nn4

### CODE REVIEW

**Comment by nicpottier** [comment]
@rowanseymour plz review

File: temba/contacts/search.py [hunk 1]
```diff
@@ -99,7 +99,6 @@ def contact_search_simple(org, query, base_queryset):
 
     return base_queryset.filter(q).distinct()
 
-
```
Comment (rowanseymour) [inline temba/contacts/search.py]: Minor point but it looks like you've added a PEP error here

**Comment by ericnewcomer** [comment]
yep

**Comment by rowanseymour** [comment]
Yup

**Pull request merged**